### PR TITLE
Initial function: is_json_valid

### PR DESCRIPTION
**Pull request by piotrek-appstream**

### CODE REVIEW

File: src/AppStream.Validation.Plugin/Json/JsonValidation.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    public JsonValidation(ILogger<JsonValidation> logger)
+    {
+        this._logger = logger;
+    }
+
+    [Function(FunctionName)]
+    [OpenApiOperation(operationId: FunctionName, tags: new[] { "ExecuteFunction" }, Description = "Checks if a string is a valid JSON.")]
+    [OpenApiParameter(name: _jsonQueryParameterName, Description = "String to be examined.", Required = true, In = ParameterLocation.Query)]
+    [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "text/plain", bodyType: typeof(string), Description = "Returns True if the string is a valid JSON, false otherwise.")]
```
Comment (lukasz-appstream) [inline src/AppStream.Validation.Plugin/Json/JsonValidation.cs]: True/false?

File: src/AppStream.Validation.Plugin/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿{
+  "AIPlugin": {
+    "SchemaVersion": "v1",
+    "NameForModel": "JSON or XML validator",
```
Comment (lukasz-appstream) [inline src/AppStream.Validation.Plugin/appsettings.json]: JSON/XML validator ?
JSON and XML validator?

Or makes sense here?

File: src/AppStream.Validation.Plugin/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "NameForHuman": "JSON or XML validator",
+    "DescriptionForModel": "Help the user validate JSONs or XMLs. You can check whether a given string is a valid JSON or XML.",
+    "DescriptionForHuman": "[ChatGPT](https://openai.com/blog/chatgpt-plugins)/[Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/plugins?tabs=Csharp) plugin for validating JSONs or XMLs.",
```
Comment (lukasz-appstream) [inline src/AppStream.Validation.Plugin/appsettings.json]: Maybe rather:
`Help the user validate JSON or XML string. You can validate format or validate it against provided schema or provided sample object`
?

File: src/AppStream.Validation.Plugin/Json/JsonValidation.cs (hunk 1 above)
Comment (piotrek-appstream) [reply src/AppStream.Validation.Plugin/Json/JsonValidation.cs]: Adjusted to the actual return value

File: src/AppStream.Validation.Plugin/appsettings.json (hunk 2 above)
Comment (piotrek-appstream) [reply src/AppStream.Validation.Plugin/appsettings.json]: Changed

File: src/AppStream.Validation.Plugin/appsettings.json (hunk 3 above)
Comment (piotrek-appstream) [reply src/AppStream.Validation.Plugin/appsettings.json]: Changed

**Pull request merged**